lib/metadata.js

```diff
@@ -22,8 +22,11 @@ function buildMetadata (model) {
 
     for (var propKey in model.entityTypes[typeKey]) {
       var property = model.entityTypes[typeKey][propKey]
-
-      entityType.Property.push({'@Name': propKey, '@Type': property.type})
+      var finalObject = { '@Name': propKey, '@Type': property.type }
+      if (property.hasOwnProperty('nullable')) {
+        finalObject = { ...finalObject, '@Nullable': property.nullable }
```

Comment:
I see this package is so far back-compatible with node 4.

So maybe you can just do this
```
var finalObject = { '@Name': propKey, '@Type': property.type }
if (property.hasOwnProperty('nullable')) {
    finalObject['@Nullable'] = property.nullable
}
```
```

Reply:
Updated to reflect those changes.  Thanks!